src/raftstore/store/store.rs

```diff
@@ -217,6 +222,7 @@ impl<T, C> Store<T, C> {
             sendch: sendch,
             significant_msg_receiver: ch.significant_msg_receiver,
             region_peers: HashMap::default(),
+            merge_states: vec![],
```

Comment:
there is a `MergeState` concept, use `merge_states` is somewhat confusing. How about use `xx_regions` instead ?